### PR TITLE
Added Galangkangin's student info and photo

### PR DESCRIPTION
**Pull request by thegalang**
Thank you for your pull request! Please read and

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request (replace the text as instructed)

I added my data to _data/students.yml, added my photo to images/students/ (galangkangin.jpg)

here is a link to the page in my forked repository: https://thegalang.github.io/gci16.fossasia.org/
here is the GCI task: https://codein.withgoogle.com/dashboard/task-instances/5595387010220032/

### CODE REVIEW

**Comment by abishekvashok** [comment]
Please update the PR body.

**Comment by thegalang** [comment]
sorry...

**Comment by abishekvashok** [comment]
Dont delete the template! Please read the readme and contributors .md files first then commit

**Comment by thegalang** [comment]
okay, i changed my body

File: _data/students.yml [hunk 1]
```diff
@@ -570,8 +570,7 @@
   twitter: guoJLbenjamin
   bio: Cynical
   lat: 1.3553794
-  lng: 103.86774439999999
-
+  lng: 103.86774
```
Comment (abishekvashok) [inline _data/students.yml:573]: Dont change others info without their consent and  add a - before the name like the other students do please

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please try to squash your commits and fix as stated please

**Comment by robbyoconnor** [comment]
I merged this in e9814d8f67d9f3d1dddcf0578eab7170650ccd48 with a follow up in fa154f0499e3ce04ef1f583f0c3dd2d62ab5d05f to fix a small syntax error.

File: _data/students.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/students.yml:573]: I may have resolved the conflict wrong...thankfully I had the old commit

**Pull request closed without merging**